contrib/minijinja-cli/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "minijinja-cli"
+pkgver = "1.0.15"
+pkgrel = 0
+build_wrksrc = "minijinja-cli"
+build_style = "cargo"
+make_build_args = ["--features=completions"]
+make_build_env = {
+    "ASSET_OUT_DIR": "assets",
+}
```

Comment:
This is intentionally only enabled for the build phase, which causes it to generate completions during the build phase. During `do_install`, cargo will build it again, but without the `completions` feature activated.

Is it okay to keep it like this? Or should I override `do_build` to be more explicit in that behaviour instead? I could explicitly add to cargo build invocations, one with the feature enabled and the env var set, and then a second one without both of these.

Reply:
i think this is ok (helps with installing the better more minimised package, and since all the steps are implicitly correct there is no need for rewriting the logic), but just add a comment like

```
# cargo install rebuilds without these args in install to not ship feature at runtime
```

etc